tdvt/tdvt/exprtests/standard/setup.calcs_data.dates.txt

```diff
@@ -0,0 +1,9 @@
+//Test the data in the table.
+date0
+date1
+date2
+date3
+datetime0
+datetime1
+time0
+time1
```

Comment:
Given `time0` and `time1` aren't actively used in tests at the moment would it make sense to split them into a setup.calcs_data.time.txt?  

A partner wanted to be able to narrowly ignore only those due to lack of TIME type support on their database I believe.

Reply:
That makes sense, I'll split them off.